cluster/config-defaults.yaml

```diff
@@ -74,6 +74,10 @@ skipper_ingress_lightstep_max_period: "2500ms"
 skipper_ingress_lightstep_log_events: ""
 lightstep_token: ""
 
+# skipper routegroup validation webhook
+validate_routegroups: "false"
```

Comment:
nit: can we use similar naming as other fields e.g. `stackset_validation_enabled`? So this would be `routegroups_validation_enabled`

Reply:
Thanks, I had hoped to get feedback on this!

Reply:
And let's enable it by default in e2e tests. See `test/e2e/cluster_config.sh`